src/clarity/forms/_forms.clarity.scss

```diff
@@ -434,6 +433,23 @@ $clr-custom-checkbox-radio-top: ($clr_baselineRem_1 - $clr-checkbox-radio-height
             display: inline-block;
         }
 
+        input[type="checkbox"]:indeterminate + label::before {
```

Comment:
I double checked that this works in IE 10 / 11: 
![ie_10_checkbox](https://cloud.githubusercontent.com/assets/433692/21058281/097580b8-bdf2-11e6-9c5f-0b8c65fffb2b.png)
![ie_11_checkbox](https://cloud.githubusercontent.com/assets/433692/21058282/0975c7c6-bdf2-11e6-8d40-8a04d36a7430.png)